server/src/main/scala/cromwell/CromwellEntryPoint.scala

```diff
@@ -93,6 +94,8 @@ object CromwellEntryPoint extends GracefulStopSupport {
       case t: Throwable =>
         Log.error("Failed to instantiate Cromwell System. Shutting down Cromwell.")
         Log.error(t.getMessage)
+        val stackTrace = ExceptionUtils.getStackTrace(t)
+        Log.error(stackTrace)
```

Comment:
I haven't tried it, but these four lines feel like `Log.error(s"Failed to instantiate Cromwell System. Shutting down Cromwell.", t)`. If however the four lines output something specific that the one line does not then this change is fine.

Reply:
I suspect you're right. 🙂